scripts/ci/get-slack-user-id.sh

```diff
@@ -58,13 +58,14 @@ case "$github_login" in
 'kylape')           slack_user='UGJML86DD'   ;;
 'ludydoo')          slack_user='U04TFDR57KQ' ;;
 'lvalerom')         slack_user='U02SJTV567N' ;;
+'mclasmeier')       slack_user='U02DKH1LQ5N' ;;
 'md2119')           slack_user='U01Q57U4JFK' ;;
 'mfosterrox')       slack_user='U01PMH71ACU' ;;
 'misberner')        slack_user='U01PFA6NUG3' ;;
 'msugakov')         slack_user='U020QJZCQAH' ;;
-'mclasmeier')       slack_user='U02DKH1LQ5N' ;;
 'mtodor')           slack_user='U039LQ48PT7' ;;
 'ovalenti')         slack_user='U03F2F9EXUL' ;;
+'parametalol')      slack_user='U02MJ72K1B5' ;;
```

Comment:
should we keep the old mapping ?

Reply:
Not 100% sure, but may be for older commits I'd keep it.